blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -282,6 +283,21 @@ public void setHttpEventsIngestionHandler(HttpEventsIngestionHandler httpEventsI
         }
     }
 
+    private void initNetty() {
+        int acceptThreads = Configuration.getInstance().getIntegerProperty(HttpConfig.MAX_WRITE_ACCEPT_THREADS);
+        int workerThreads = Configuration.getInstance().getIntegerProperty(HttpConfig.MAX_WRITE_WORKER_THREADS);
+
+        if ( NettyUtils.get().isEpollAvailable() ) {
```

Comment:
Do you think, it can be beneficial to have a feature flag to control this behavior? Think of a scenario where we just want to turn off native epoll, even when its available.